DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs

```diff
@@ -43,7 +43,7 @@ public override Task CreateAsync(AuditTrailCreateContext context)
              *  Cancel                      - N/A (exits page)
              */
 
-            if (context is AuditTrailCreateContext<AuditTrailContentEvent> contentEvent)
+            if (context is AuditTrailCreateContext<AuditTrailContentEvent> contentEvent && !contentEvent.Name.Equals("Created", StringComparison.InvariantCultureIgnoreCase))
```

Comment:
Could perhaps use/add to constants for "Created" like Constants.ContentEvent_xxxx

Reply:
fixed and pushed